assets/css/main.css

```diff
@@ -1 +1,55 @@
-/* custom styles */
\ No newline at end of file
+/* custom styles */
+h1 {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 700;
+  font-size: 50px;
+  text-transform: uppercase;
+  text-align: center;
+
+}
+nav {
+  font-size: 15px;
+  text-align: center;
+  text-transform: uppercase;
+  font-weight: 500;
+  background: pink;
+  padding: 15px 0;
+}
+a {
+  text-decoration: none;
+  color: white;
+  padding: 0 10px;
+}
+h2 {
```

Comment:
Consider combining your heading styles so that there's not so much repetition in your code:

For example:
```css
h1, h2 {
  font-family: 'Open Sans', sans-serif;
  font-weight: 700;
  font-size: 50px;
  text-transform: uppercase;
  text-align: center;
}

h2 {
  font-size: 20px
}
```

You also use Open-Sans a lot, what if you set it as your default on the body?

For example:
```css
body {
  font-family: 'Open Sans', sans-serif;
}
```